google-cloud-storage/src/main/java/com/google/cloud/storage/StorageImpl.java

```diff
@@ -543,6 +547,32 @@ public ReadChannel reader(BlobId blob, BlobSourceOption... options) {
     return new BlobReadChannel(getOptions(), blob, optionsMap);
   }
 
+  @Override
+  public void downloadTo(BlobId blob, Path path, Blob.BlobSourceOption... options) {
```

Comment:
Oh, you did get the method added here. Can you update `Blob#downloadTo(Path, BlobSourceOption...)` to call this?